spec/features/sdr_deposit_spec.rb

```diff
@@ -14,16 +14,15 @@
   end
 
   it 'deposits objects' do
-    ensure_token
```

Comment:
No need to sprinkle the token code all over since it's handled within the client now.